### PR TITLE
[DOC-7931][DOC-8902] CockroachDB Dedicated on Azure is GA

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-7931] CockroachDB Dedicated on Azure is GA
[DOC-8902] Document how IOPS works on Azure

Also found a few things we missed in last week's release.

### Previews
[releases/cloud.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/releases/cloud.html)
[src/current/cockroachcloud/architecture.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/architecture.html)
[src/current/cockroachcloud/cloud-api.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/cloud-api.html)
[src/current/cockroachcloud/cluster-management.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/cluster-management.html)
[src/current/cockroachcloud/cmek.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/cmek.html)
[src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/cockroachdb-dedicated-on-azure.html)
[src/current/cockroachcloud/connect-to-your-cluster.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-your-cluster.html)
[src/current/cockroachcloud/create-your-cluster.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-your-cluster.html)
[src/current/cockroachcloud/egress-perimeter-controls.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/egress-perimeter-controls.html)
[src/current/cockroachcloud/frequently-asked-questions.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/frequently-asked-questions.html)
[src/current/cockroachcloud/managing-cmek.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/managing-cmek.html)
[src/current/cockroachcloud/network-authorization.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/network-authorization.html)
[src/current/cockroachcloud/pci-dss.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/pci-dss.html)
[src/current/cockroachcloud/private-clusters.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/private-clusters.html)
[src/current/cockroachcloud/provision-a-cluster-with-terraform.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/provision-a-cluster-with-terraform.html)
[src/current/cockroachcloud/regions.md](https://deploy-preview-17931--cockroachdb-docs.netlify.app/docs/cockroachcloud/regions.html)

### CODE REVIEW

File: src/current/cockroachcloud/regions.md [hunk 1]
```diff
@@ -77,10 +77,6 @@ Western Europe  | `eu-central-1`   | Frankfurt
 
 ## Azure regions
 
-{{site.data.alerts.callout_info}}
-{% include feature-phases/azure-limited-access.md %}
```
Comment (adwittumuluri) [inline src/current/cockroachcloud/regions.md:81]: Can [`azure-limited-access.md`](https://github.com/cockroachdb/docs/blob/main/src/current/_includes/feature-phases/azure-limited-access.md) be removed entirely, now?

File: src/current/cockroachcloud/create-your-cluster.md [hunk 2]
```diff
@@ -95,7 +83,9 @@ The choice of hardware per node determines the [cost](#step-2-select-the-cloud-p
     Storage space cannot be removed due to cloud provider limitations.
     {{site.data.alerts.end}}
 
-    For optimal performance, choose up to <b>{{ cap_per_vcpu }}</b>. Refer to [Pricing](https://www.cockroachlabs.com/pricing/) for details. When selecting your storage capacity, consider the following factors:
+    For optimal performance, choose up to <b>{{ cap_per_vcpu }}</b>. Refer to [Pricing](https://www.cockroachlabs.com/pricing/) for details. IOPS for Azure disks are determined by the disk size, regardless of the VM configuration. To learn more, refer to [Premium SSD size](https://learn.microsoft.com/azure/virtual-machines/disks-types#premium-ssd-size) in the Azure documentation.
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-your-cluster.md:86]: @fantapop PTAL

**Review by adwittumuluri — APPROVED** [review]
I think i'll need to go over the rendered docs once this is in, just to make sure we didn't miss anything. I see we missed some multi-region related things ([example](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster#step-3-select-the-region-s)) in previous changes.

**Comment by mdlinville** [comment]
> example

I think I got them all. All the previews are available for your review now.

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 3]
```diff
@@ -90,7 +84,7 @@ Yes, existing CockroachDB {{ site.data.products.cloud }} customers can pay for t
 
 ### Does the CockroachDB {{ site.data.products.cloud }} technical SLA cover clusters on Azure?
 
-During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), the [CockroachDB {{ site.data.products.cloud }} technical SLA](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) does not apply to CockroachDB {{ site.data.products.dedicated }} clusters on Azure. For more details about the roadmap, contact your Cockroach Labs account team.
```
Comment (kannanlakshmi) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:93]: same comment as above, consider removing the whole question as its not unique to Azure.

**Review by kannanlakshmi — APPROVED** [review]
LGTM - very minor nits - thank you!

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 4]
```diff
@@ -62,11 +58,9 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
 The following sections provide more details about CockroachDB {{ site.data.products.dedicated }} on Azure.
 
-### What is Limited Access and how does it relate to CockroachDB {{ site.data.products.dedicated }} on Azure?
-
-CockroachDB {{ site.data.products.dedicated }} on Azure is fully managed, like on GCP or AWS. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), your {{ site.data.products.db }} organization must be enrolled before you can deploy CockroachDB {{ site.data.products.dedicated }} clusters on Azure.
+### Is CockroachDB {{ site.data.products.dedicated }} on Azure generally available?
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md]: Done

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md (hunk 3 above)
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:93]: Done

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 5]
```diff
@@ -6,15 +6,11 @@ toc_not_nested: true
 docs_area: deploy
 ---
 
-This page provides information about CockroachDB {{ site.data.products.dedicated }} clusters on Microsoft Azure, including frequently asked questions and limitations during [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability). To create a cluster, refer to [Create Your Cluster]({% link cockroachcloud/create-your-cluster.md %}).
-
-{{site.data.alerts.callout_info}}
-{% include feature-phases/azure-limited-access.md %}
-{{site.data.alerts.end}}
+This page provides information about CockroachDB {{ site.data.products.dedicated }} clusters on Microsoft Azure, including frequently asked questions and limitations. To create a cluster, refer to [Create Your Cluster]({% link cockroachcloud/create-your-cluster.md %}).
```
Comment (gemma-shay) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:9]:
```suggestion
This page provides information about CockroachDB {{ site.data.products.dedicated }} clusters on Microsoft Azure, including frequently asked questions and limitations. To create a cluster, refer to [Create a CockroachDB {{ site.data.products.dedicated }} Cluster]({% link cockroachcloud/create-your-cluster.md %}).
```

File: src/current/cockroachcloud/create-your-cluster.md [hunk 6]
```diff
@@ -28,21 +28,17 @@ CockroachDB {{ site.data.products.dedicated }} advanced clusters have access to
 
 ## Step 2. Select the cloud provider
 
-In the **Cloud provider** section, select your deployment environment: **Google Cloud**, **AWS**, or **Microsoft Azure** (Limited Access).
+In the **Cloud provider** section, select your deployment environment: **Google Cloud**, **AWS**, or **Microsoft Azure**.
```
Comment (gemma-shay) [inline src/current/cockroachcloud/create-your-cluster.md:31]: Suggest mentioning here or in the step right above that there's no azure option for dedicated advanced

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 7]
```diff
@@ -6,15 +6,11 @@ toc_not_nested: true
...
 
 ## Limitations
 
-During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
+CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
 
 ### Regions
```
Comment (gemma-shay) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:15]: This appears to be slightly different from the regions page documentation on azure. Suggest using an include or linking out

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 8]
```diff
@@ -62,12 +58,6 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
 The following sections provide more details about CockroachDB {{ site.data.products.dedicated }} on Azure.
 
-### What is Limited Access and how does it relate to CockroachDB {{ site.data.products.dedicated }} on Azure?
-
-CockroachDB {{ site.data.products.dedicated }} on Azure is fully managed, like on GCP or AWS. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), your {{ site.data.products.db }} organization must be enrolled before you can deploy CockroachDB {{ site.data.products.dedicated }} clusters on Azure.
-
-The clusters created during this period are recommended for proof-of-concept and testing, and are not suitable for production. The [CockroachDB {{ site.data.products.cloud }} Service Level Agreement (SLA)](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) is not applicable to Azure clusters during limited access. Azure clusters are excluded from premium support agreements during limited access, and technical support is only available during business hours.
-
 ### Are multi-region clusters supported?
```
Comment (gemma-shay) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:61]: Suggest adding that Serverless clusters are not supported

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md [hunk 9]
```diff
@@ -6,15 +6,11 @@ toc_not_nested: true
 docs_area: deploy
```
Comment (gemma-shay) [inline src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:6]: Maybe you are already planning for this, but I think as this page is pared down we can probably move all this info to an Azure section of the cloud FAQS

**Review by gemma-shay — APPROVED** [review]

**Comment by mdlinville** [comment]
Lakshmi approved offline.

File: src/current/cockroachcloud/create-your-cluster.md (hunk 6 above)
Comment (mdlinville) [inline src/current/cockroachcloud/create-your-cluster.md:31]:
```suggestion
In the **Cloud provider** section, select your deployment environment: **Google Cloud**, **AWS**, or **Microsoft Azure**.

{{site.data.alerts.callout_info}}
CockroachDB {{ site.data.products.dedicated }} advanced clusters cannot currently be deployed on Azure.
{{site.data.alerts.end}}

```

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md (hunk 9 above)
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:6]: That's a good point, but I don't think we should do it quite yet, until a few more of the limitations are lifted.

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md (hunk 7 above)
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:15]: Removed the specific list here and pointed to the main Regions topic.

File: src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md (hunk 8 above)
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md:61]: Good idea, done.

**Pull request merged**